vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoLightTheme.java

```diff
@@ -32,39 +32,38 @@
 public class LumoLightTheme extends Theme {
 
     protected static final int BORDER_RADIUS = 5;
```

Comment:
I assume this is pixels. The `--lumo-border-radius-m` property is by default `0.25rem` which equals `4px` with browser default styles. Though, I’m not sure where this property is used, if it makes sense to align it.

```suggestion
    protected static final int BORDER_RADIUS = 4;
```